### PR TITLE
Club: adding readme file

### PR DESCRIPTION
**Pull request by matiasbenedetto**
#### Changes proposed in this Pull Request:
Club: adding readme file

### CODE REVIEW

File: club/readme.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
+=== Club ===
+Contributors: Automattic
+Requires at least: 5.8
+Tested up to: 6.0.1
+Requires PHP: 5.7
+License: GPLv2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+
+A blank starting point for building your site.
```
Comment (mikachan) [inline club/readme.txt]: I don't think we need this line.

**Review by mikachan — COMMENTED** [review]
Looks good!

We should add a `== Changelog ==` section too, something like this:

```
== Changelog ==

= 1.0.0 =
* Initial release
```

File: club/readme.txt (hunk 1 above)
Comment (matiasbenedetto) [reply club/readme.txt]: removed!

**Review by mikachan — APPROVED** [review]
LGTM!

**Pull request merged**